x-pack/test/functional/apps/transform/actions/reauthorizing.ts

```diff
@@ -0,0 +1,214 @@
+        originalConfig: getLatestTransformConfig(`${PREFIX}-${USER.TRANSFORM_POWERUSER}-1`, true),
+        mode: 'continuous',
+        type: 'latest',
+        expected: {
+          originalState: { status: TRANSFORM_STATE.STARTED, health: TRANSFORM_HEALTH_LABEL.green },
+          reauthorizeEnabled: false,
+          reauthorizedState: {
+            status: TRANSFORM_STATE.STARTED,
+            health: TRANSFORM_HEALTH_LABEL.green,
+          },
+        },
+        created_by_user: USER.TRANSFORM_POWERUSER,
+        current_user: USER.TRANSFORM_VIEWER,
+      },
+    ];
+
+    before(async () => {
+      const apiKeyForTransformUsers =
+        await transform.securityCommon.createApiKeyForTransformUsers();
+
+      apiKeyForTransformUsers.forEach(({ user, apiKey }) =>
+        apiKeysForTransformUsers.set(user.name as USER, apiKey)
+      );
+
+      await esArchiver.loadIfNeeded('x-pack/test/functional/es_archives/ml/ecommerce');
+      await transform.testResources.createIndexPatternIfNeeded('ft_ecommerce', 'order_date');
+
+      for (const testData of testDataList) {
+        await transform.api.createTransform(testData.originalConfig.id, testData.originalConfig, {
+          deferValidation: true,
+          // Create transforms with secondary authorization headers
+          headers: generateHeaders(apiKeysForTransformUsers.get(testData.created_by_user)!),
+        });
+        // For transforms created insufficient permissions, they can be created but not started
+        // so we should not assert that the api call is successful here
+        await transform.api.startTransform(testData.originalConfig.id, false);
+      }
+      await transform.testResources.setKibanaTimeZoneToUTC();
+    });
+
+    after(async () => {
+      await transform.securityCommon.clearAllTransformApiKeys();
+
+      for (const testData of testDataList) {
+        await transform.testResources.deleteIndexPatternByTitle(testData.originalConfig.dest.index);
+        await transform.api.deleteIndices(testData.originalConfig.dest.index);
+      }
+
+      await transform.api.cleanTransformIndices();
+      await transform.testResources.deleteIndexPatternByTitle('ft_ecommerce');
+    });
+
+    for (const testData of testDataList) {
+      const transformId = testData.originalConfig.id;
+
+      describe(`${testData.suiteTitle}`, function () {
+        it('reauthorize transform', async () => {
+          await transform.securityUI.loginAs(testData.current_user);
+
+          await transform.testExecution.logTestStep('should load the home page');
+          await transform.navigation.navigateTo();
+          await transform.management.assertTransformListPageExists();
+
+          await transform.testExecution.logTestStep('should display the transforms table');
+          await transform.management.assertTransformsTableExists();
+
+          await transform.testExecution.logTestStep('should display the transforms table');
```

Comment:
Is the text a copy and paste here? Shouldn't it be `should display the transforms reauthorize callout` ?